files/kubelet-kubeconfig

```diff
@@ -16,10 +16,11 @@ users:
   user:
     exec:
       apiVersion: client.authentication.k8s.io/v1alpha1
-      command: /usr/bin/aws-iam-authenticator
+      command: aws
       args:
-        - "token"
-        - "-i"
-        - "CLUSTER_NAME"
         - --region
-        - "AWS_REGION"
\ No newline at end of file
+        - "AWS_REGION"
+        - eks
+        - get-token
```

Comment:
will aws eks get-token use regional sts for us-west-2?
if not, consider add an env `AWS_STS_REGIONAL_ENDPOINTS=regional ` and test it. you should find the sts endpoint in the token itself.

Reply:
Looks like it uses the IAD Endpoint. Will set that environment variable in the Kubeconfig to redirect to regional endpoint.